Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/Controls/InfiniteCanvasVirtualDrawingSurface.Render.cs

```diff
@@ -66,7 +66,9 @@ internal void ReDraw(Rect viewPort)
                 toDraw.Width = Width;
             }
 
-            using (CanvasDrawingSession drawingSession = CanvasComposition.CreateDrawingSession(_drawingSurface, toDraw))
+            var scale = _screenScale * zoom;
+
+            using (CanvasDrawingSession drawingSession = CanvasComposition.CreateDrawingSession(_drawingSurface, ScaleRect(toDraw, scale), 96 * (float)scale))
```

Comment:
Should this `96` be grabbed from the display info?

Reply:
I put it into a named constant. The value is not dependent on the actual display DPI. It is the internal default DPI of Win2D. [Link](https://microsoft.github.io/Win2D/html/M_Microsoft_Graphics_Canvas_UI_Composition_CanvasComposition_CreateDrawingSession_2.htm)